CMakeLists.txt

```diff
@@ -25,9 +25,9 @@ option(BUILD_TESTS "Build tests" ON)
 option(BUILD_PYTHON "Build Python SWIG module" ON)
 option(BUILD_DOC "Build Documentation" OFF)
 option(BUILD_SHARED_LIBS "Build shared library(.so)" ON)
-option(USE_ORTOOLS_RELEASE "Get or-tools from release tarball" ON)
+option(USE_ORTOOLS_RELEASE "Get or-tools from release tarball" OFF)
 option(USE_ORTOOLS_FETCH_SRC "Get or-tools from source" OFF)
-option(USE_ORTOOLS_VENDOR "Get or-tools from ortools_vendor" OFF)
+option(USE_ORTOOLS_VENDOR "Get or-tools from ortools_vendor" ON)
```

Comment:
To behave as a ROS package built from source that is easily used by colcon builds in a wider ros environment we need to default USE_ORTOOLS_VENDOR to ON.

If you don't want that, then the option needs to at least be set on for ROS2 packaging builds. i.e. Like it was done for FETCHCONTENT_FULLY_DISCONNECTED=OFF
https://github.com/ros2-gbp/fields2cover-release/blob/patches/debian/rolling/fields2cover/0002-Customized-debian-rules-file.patch  

I don't know how you actually did that.  Did you have to visit all the active branches?